Exercises/Celina/exercises_06.py

```diff
@@ -0,0 +1,165 @@
+#       - i (file, size=584)
+#     - f (file, size=29116)
+#     - g (file, size=2557)
+#     - h.lst (file, size=62596)
+#   - b.txt (file, size=14848514)
+#   - c.dat (file, size=8504156)
+#   - d (dir)
+#     - j (file, size=4060174)
+#     - d.log (file, size=8033020)
+#     - d.ext (file, size=5626152)
+#     - k (file, size=7214296)
+#
+#
+# PART 1:
+# Write a function that reads a console session like the example above from a file, 
+# and then returns a dictionary listing the size of each directory. Directories should
+# be identified by their full path. For the example above:
+#
+# - The total size of directory `e` is 584 because it contains a single file `i` of size 584 and no other directories.
+# - The directory `a` has total size 94853 because it contains files
+#     `f` (size 29116), `g` (size 2557), and `h.lst` (size 62596),
+#     plus file `i` indirectly (`a` contains `e` which contains `i`).
+# - Directory `d` has total size 24933642.
+# - As the outermost directory, `/` contains every file. Its total size is 48381165.
+#
+# Based on that functions output, calculate the total size of all directories where the size of each individual
+# directory considered is at most 100000, and mention the result in your PR.
+# For the example above the two directories meeting the size requirement are `a` and `e`, while `/` and `d` are too large.
+# The sum of the size of a and e would be 95437.
+#
+#
+# PART 2:
+# In part 2 you need to identify a single directory to delete (including all sub-directories of course).
+# The total space available to the filesystem is 70000000, and you need to make enough room to fit a file of 
+# size 30000000.
+#
+# In the example above, the total size of the outermost directory (and thus the total amount of used space) is 48381165; 
+# this means that the size of the unused space must currently be 21618835, which isn't quite the 30000000 required.
+# Therefore, the update still requires a directory with total size of at least 8381165 to be deleted before it can run.
+#
+# To achieve this, you have the following options:
+#
+# Delete directory `e`, which would increase unused space by 584.
+# Delete directory `a`, which would increase unused space by 94853.
+# Delete directory `d`, which would increase unused space by 24933642.
+# Delete directory `/`, which would increase unused space by 48381165.
+# Directories `e` and `a` are both too small; deleting them would not free up enough space.
+# However, directories `d` and `/` are both big enough! 
+# Between these, choose the smallest: `d`, increasing unused space by 24933642.
+# 
+# Mention the total size of the deleted directory in the PR.
+# 
+# You can find the /actual/ input to both parts in data/terminal_record.txt
+###############################################################################
+
+
+file_path = []
+
+directory_dic={}
+
+
+console_session = open('terminal_record.txt', 'r')
+input = []
+for line in console_session:
+    input.append(line)
```

Comment:
you should always close a file that your opening. Consider doing
```suggestion
with open('terminal_record.txt', 'r') as console_session:
    input = []
    for line in console_session:
        input.append(line)
```
that way the file will automatically be closed
Also having to copy the terminal_record.txt into the working directory is not a nice solution, although its functioning. you could use pathlib to use a relative path.